marshal.c

```diff
@@ -2375,5 +2373,5 @@ rb_marshal_dump(VALUE obj, VALUE port)
 VALUE
 rb_marshal_load(VALUE port)
 {
-    return rb_marshal_load_with_proc(port, Qnil);
+    return rb_marshal_load_with_proc(port, Qnil, Qfalse);
```

Comment:
Passing `Qfalse` as a `bool` may cause a "shortening" warning.
```suggestion
    return rb_marshal_load_with_proc(port, Qnil, false);
```